doc/de/Install.md

```diff
@@ -4,90 +4,132 @@ Friendica Installation
 
 Bevor du anfängst: suche dir einen Domain- oder Subdomainnamen für deinen Server.
 Dinge verändern sich und einige deiner Freunde haben möglicherweise Probleme, mit dir zu kommunizieren.
 Wir planen, diese Einschränkung in einer zukünftigen Version zu beheben.
 
 
-1. Voraussetzungen
-    - Apache mit einer aktiverten mod-rewrite-Funktion und dem Eintrag "Options All", so dass du die lokale .htaccess-Datei nutzen kannst
-    - PHP  5.6+. Je neuer, desto besser.
-        - PHP *Kommandozeilen*-Zugang mit register_argc_argv auf "true" gesetzt in der php.ini-Datei
-        - Curl, GD, PDO, MySQLi, xml, zip und OpenSSL-Erweiterung
-        - etwas in der Art eines Email-Servers oder eines Gateways wie PHP mail()
-        - Das POSIX Modul muss aktiviert sein ([CentOS, RHEL](http://www.bigsoft.co.uk/blog/index.php/2014/12/08/posix-php-commands-not-working-under-centos-7http://www.bigsoft.co.uk/blog/index.php/2014/12/08/posix-php-commands-not-working-under-centos-7) haben dies z.B. deaktiviert)
-    - Mysql 5.5.3+
-    - die Möglichkeit, wiederkehrende Aufgaben mit cron (Linux/Mac) oder "Scheduled Tasks" einzustellen (Windows) [Beachte: andere Optionen sind in Abschnitt 7 dieser Dokumentation zu finden]
-    - Installation in einer Top-Level-Domain oder Subdomain (ohne eine Verzeichnis/Pfad-Komponente in der URL) wird bevorzugt. Verzeichnispfade sind für diesen Zweck nicht so günstig und wurden auch nicht ausführlich getestet.
+Requirements
+---
 
+* Apache mit einer aktiverten mod-rewrite-Funktion und dem Eintrag "Options All", so dass du die lokale .htaccess-Datei nutzen kannst
+* PHP  5.6+ (PHP 7 ist aufgrund der Performance empfohlen)
+  * PHP *Kommandozeilen*-Zugang mit register_argc_argv auf "true" gesetzt in der php.ini-Datei
+  * Curl, GD, PDO, MySQLi, xml, zip und OpenSSL-Erweiterung
+  * Das POSIX Modul muss aktiviert sein ([CentOS, RHEL](http://www.bigsoft.co.uk/blog/index.php/2014/12/08/posix-php-commands-not-working-under-centos-7http://www.bigsoft.co.uk/blog/index.php/2014/12/08/posix-php-commands-not-working-under-centos-7) haben dies z.B. deaktiviert)
+  * etwas in der Art eines Email-Servers oder eines Gateways wie PHP mail()
+* Mysql 5.5.3+ (oder eine äquivalente Alternative: MariaDB, Percona Server etc.)
+* die Möglichkeit, wiederkehrende Aufgaben mit cron (Linux/Mac) oder "Scheduled Tasks" einzustellen (Windows) [Beachte: andere Optionen sind in Abschnitt 7 dieser Dokumentation zu finden]
+* Installation in einer Top-Level-Domain oder Subdomain (ohne eine Verzeichnis/Pfad-Komponente in der URL) wird bevorzugt. Verzeichnispfade sind für diesen Zweck nicht so günstig und wurden auch nicht ausführlich getestet.
 
-    [Dreamhost.com bietet ein ausreichendes Hosting-Paket mit den nötigen Features zu einem annehmbaren Preis. Wenn dein Hosting-Anbieter keinen Unix-Zugriff erlaubt, kannst du Schwierigkeiten mit der Einrichtung der Webseite haben.
+Installation procedure
```

Comment:
EN in the DE docs? This should be translated as well.

Reply:
I'll admit I took @M-arcus word for it when he said he was going to translate 😅

Reply:
Fixed here https://github.com/friendica/friendica/pull/4916